script/Ciel/Day 9/09_01_CIEL09_1C.txt

```diff
@@ -142,7 +156,8 @@ I enter the infirmary.
 [sha:2f77f4394acb5a064eb94a94d71e1b8b0cbb2f10]{
 -- Page 64, Offset 22298.
 -- 見慣れた保健室には、やはり、学校に死ぬほどそぐわないファンキーな医師の姿があった。
-Inside this familiar room, I see, as I expected, a funky doctor, terribly unsuited to a school setting.
+// There is no way you can convince me to directly transliterate ファンキー here
+As I'd feared, the familiar sight of the infirmary has been disrupted by a kooky doctor that looks utterly out of place in a school setting.
```

Comment:
>As I'd feared, the familiar sight of the infirmary has been disrupted by a kooky doctor so at odds with the school setting that it couldn't be more unfitting.

The last clause should maybe be a bit more interesting?

Reply:
i like your suggestion but comma after doctor

Reply:
"it couldn't be more unfitting" feels like a bit of an awkward construction to me and doesn't really add much interest beyond 'utterly'. If you want to get more creative maybe sth along the lines of?
> doctor who clashes garishly with the school setting.
> doctor who fits in with the school setting about as well as a bull does a china shop.